scripts/Muon/GUI/ElementalAnalysis/PeriodicTable/PeakSelector/peak_selector_view.py

```diff
@@ -35,22 +35,31 @@ def __init__(self, peak_data, element, parent=None):
         if not valid_data(peak_data):
             raise ValueError
 
-        self.update_new_data(peak_data)
         self.element = element
+        self.update_new_data(peak_data)
         self.setWindowTitle(element)
         self.list = QtWidgets.QVBoxLayout(self)
 
-        # Labels might not be present, if so return empty list
+        # Gamma peaks might not be present, if so return empty list
         primary = peak_data["Primary"]
         self.primary_checkboxes = self._create_checkbox_list("Primary", primary)
         secondary = peak_data["Secondary"]
         self.secondary_checkboxes = self._create_checkbox_list("Secondary", secondary, checked=False)
         try:
             gammas = peak_data["Gammas"]
-            self.gamma_checkboxes = self._create_checkbox_list(
-                "Gammas", gammas, checked=False)
+            self.gamma_checkboxes = self._create_checkbox_list("Gammas", gammas, checked=False)
         except KeyError:
             self.gamma_checkboxes = []
+        try:
+            # Electron data has the x position as key and relative intensity as value
+            electrons = peak_data["Electrons"]
+            electron_data = {}
+            for xpos, int in electrons.items():
+                name = 'e- {}'.format(xpos)
```

Comment:
we need more of a space between e- and the number. I think you should be able to do $e^-$ to get a nice electron symbol

Reply:
```suggestion
                name = '$e^-\quad$  {}'.format(xpos)
```